README.md

```diff
@@ -1191,6 +1192,11 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 
   * [serverlesstechnology/cqrs](https://github.com/serverlesstechnology/cqrs) [[cqrs-es](https://crates.io/crates/cqrs-es)] — A framework for CQRS and event sourcing with [user guide](https://doc.rust-cqrs.org/)
 
+### EBPF
```

Comment:
Why the capitalisation? Shouldn't this be "eBPF"?

Reply:
@palfrey eBPF is an unequivocally recognized acronym. Would the linter approve of a sentence that begins with a lowercase letter?

Reply:
I believe so. It's mostly checking "does the top-level ToC equal the actual sections".